com.unity.render-pipelines.high-definition/Runtime/Lighting/Light/HDAdditionalLightData.cs

```diff
@@ -2678,6 +2682,18 @@ void LateUpdate()
                 timelineWorkaround.oldDisplayAreaLightEmissiveMesh = displayAreaLightEmissiveMesh;
                 timelineWorkaround.oldLightColorTemperature = legacyLight.colorTemperature;
             }
+
+            if (timelineWorkaround.renderEntityDataVersion != m_CurrentRenderEntityVersion)
+            {
+                timelineWorkaround.renderEntityDataVersion = m_CurrentRenderEntityVersion;
+                UpdateRenderEntity();
+            }
+        }
+
+
+        void DirtyRenderEntityData()
+        {
+            ++m_CurrentRenderEntityVersion;
         }
 
         void OnDidApplyAnimationProperties()
```

Comment:
A bit nit-picky, but is this "fromTimeline" stuff actually necessarily from timeline? It could be any playable graph, including MecAnim, no?